src/test/rustdoc/issue-80893.rs

```diff
@@ -0,0 +1,7 @@
+// compile-flags: --test -Z unstable-options --test-builder true --runtool true
+
+/// ```
+/// This does not compile, but specifying a custom --test-builder should let this pass anyway
+/// `true` does not generate an output file to run, so we also specify it as a runtool
```

Comment:
You can avoid dealing with runtool by setting this as `no_run`.

```suggestion
// compile-flags: --test -Z unstable-options --test-builder true

/// ```no_run
/// This tests that `--test-builder` is accepted as a flag by rustdoc.
/// ```

Reply:
Oh, duh. That's way better.